pillar/build-infra/build-logs.sls

```diff
@@ -1,7 +1,13 @@
 
 build-infra:
+# public gpg key associated to the one used to sign log commits
+# it allows to not rely on gpg key servers
+  build_log_public_key: |
+    -----BEGIN PGP PUBLIC KEY BLOCK-----
+    ...
+    -----END PGP PUBLIC KEY BLOCK-----
 # ssh key used to access github - needs to have write access to build-logs repository
   github_ssh_key: |
-    -----BEGIN RSA PRIVATE KEY-----
+    -----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Is this change intentional? Since the key is put in `id_rsa`, I think it is expected to be RSA.

Reply:
I thought it was default to have OPENSSH. But well, it seems not.